homeassistant/components/unifiprotect/camera.py

```diff
@@ -143,12 +142,7 @@ def _async_update_device_from_protect(self) -> None:
         self._attr_is_recording = self.device.is_connected and self.device.is_recording
 
         self._async_set_stream_source()
-
-    @callback
-    def _async_update_extra_attrs_from_protect(self) -> dict[str, Any]:
-        """Add additional Attributes to Camera."""
-        return {
-            **super()._async_update_extra_attrs_from_protect(),
+        self._attr_extra_state_attributes = {
             ATTR_WIDTH: self.channel.width,
```

Comment:
Unless they need this data for automations, we shouldn't store it in the state machine.

Reply:
The data is actually _not_ static. Only channel ID is static and that is mostly just for the user to map it back to UniFi protect themselves. The width/height/bitrate/FPS and call change in a number of cases.

Reply:
Is it interesting to automate on this info?

Reply:
Maybe we can just log it, if it's good to have for troubleshooting?

Reply:
The information _should_ not change without user initiating the change, so possibly not? But it could be useful for things like generating snapshots or notification automations if you just need a reference value since they are not fixed:

* FPS is a user selectable slider. Also affected by "video mode"
* Bitrate is user selectable slider (not directly, but a "quality" value from best to worse
* Height/Width will be based on the camera and the "video mode", "4K" cameras become "2K' or "1080p" if you turn on the high FPS video mode.

Reply:
I played around with the protect UI a bit.  I agree this is OK.